dts/common/nordic/nrf54l15.dtsi

```diff
@@ -0,0 +1,651 @@
+							<8 NRF_DEFAULT_IRQ_PRIORITY>,
+							<9 NRF_DEFAULT_IRQ_PRIORITY>,
+							<10 NRF_DEFAULT_IRQ_PRIORITY>,
+							<11 NRF_DEFAULT_IRQ_PRIORITY>,
+							<12 NRF_DEFAULT_IRQ_PRIORITY>,
+							<13 NRF_DEFAULT_IRQ_PRIORITY>,
+							<14 NRF_DEFAULT_IRQ_PRIORITY>,
+							<15 NRF_DEFAULT_IRQ_PRIORITY>,
+							<16 NRF_DEFAULT_IRQ_PRIORITY>,
+							<17 NRF_DEFAULT_IRQ_PRIORITY>,
+							<18 NRF_DEFAULT_IRQ_PRIORITY>,
+							<19 NRF_DEFAULT_IRQ_PRIORITY>,
+							<20 NRF_DEFAULT_IRQ_PRIORITY>,
+							<21 NRF_DEFAULT_IRQ_PRIORITY>,
+							<22 NRF_DEFAULT_IRQ_PRIORITY>,
+							<23 NRF_DEFAULT_IRQ_PRIORITY>,
+							<24 NRF_DEFAULT_IRQ_PRIORITY>,
+							<25 NRF_DEFAULT_IRQ_PRIORITY>,
+							<26 NRF_DEFAULT_IRQ_PRIORITY>,
+							<27 NRF_DEFAULT_IRQ_PRIORITY>,
+							<28 NRF_DEFAULT_IRQ_PRIORITY>,
+							<29 NRF_DEFAULT_IRQ_PRIORITY>,
+							<30 NRF_DEFAULT_IRQ_PRIORITY>,
+							<31 NRF_DEFAULT_IRQ_PRIORITY>;
+				#mbox-cells = <1>;
+				nordic,tasks = <18>;
+				nordic,tasks-mask = <0x007f0000>;
+			};
+		};
+	};
+
+	clocks {
+		lfxo: lfxo {
+			compatible = "nordic,nrf-lfxo";
+			#clock-cells = <0>;
+			clock-frequency = <32768>;
+		};
+
+		hfxo: hfxo {
+			compatible = "nordic,nrf-hfxo";
+			#clock-cells = <0>;
+			clock-frequency = <DT_FREQ_M(32)>;
+		};
+	};
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		uicr: uicr@ffd000 {
+			compatible = "nordic,nrf-uicr";
+			reg = <0xffd000 0x1000>;
+		};
+
+		ficr: ficr@ffc000 {
+			compatible = "nordic,nrf-ficr";
+			reg = <0xffc000 0x1000>;
+			#nordic,ficr-cells = <1>;
+		};
+
+		cpuapp_sram: memory@20000000 {
+			compatible = "mmio-sram";
+			reg = <0x20000000 DT_SIZE_K(188)>;
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x20000000 0x2f000>;
+		};
+
+		cpuflpr_sram: memory@2002f000 {
+			compatible = "mmio-sram";
+			reg = <0x2002f000 DT_SIZE_K(68)>;
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x2002f000 0x11000>;
+		};
+
+		global_peripherals: peripheral@50000000 {
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x50000000 0x10000000>;
+
+			dppic00: dppic@42000 {
+				compatible = "nordic,nrf-dppic";
+				reg = <0x42000 0x808>;
+				status = "disabled";
+			};
+
+			spi00: spi@4a000 {
+				/*
+				 * This spi node can be either SPIM or SPIS,
+				 * for the user to pick:
+				 * compatible = "nordic,nrf-spim" or
+				 *              "nordic,nrf-spis".
+				 */
+				compatible = "nordic,nrf-spim";
+				#address-cells = <1>;
+				#size-cells = <0>;
+				reg = <0x4a000 0x1000>;
+				interrupts = <74 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-frequency = <DT_FREQ_M(32)>;
+				easydma-maxcnt-bits = <16>;
+				rx-delay-supported;
+				rx-delay = <1>;
+				status = "disabled";
+			};
+
+			uart00: uart@4a000 {
+				compatible = "nordic,nrf-uarte";
+				reg = <0x4a000 0x1000>;
+				interrupts = <74 NRF_DEFAULT_IRQ_PRIORITY>;
+				status = "disabled";
+			};
+
+			cpuflpr_vpr: vpr@4c000 {
```

Comment:
Please add entries for `cpuflpr_vpr` and `cpuflpr_clic` in `validate_base_addresses.c`.